docs/howto/sample_data.md

```diff
@@ -0,0 +1,64 @@
+# Use elastic-package to generate sample data
+
+`elastic-package` with integrations can be used to generate sample data for integration packages. The following is a quick guide on how to use it with certain packages with your own stack deployment.
+
+## Getting started
+
+### Install elastic-package
+
+The first thing needed, is an installation of `elastic-package`. If you have go installed you can run `go install github.com/elastic/elastic-package@latest`. Otherwise you find [here](https://github.com/elastic/elastic-package/tree/main?tab=readme-ov-file#getting-started) different ways how to install `elastic-package.
+
+### Setup Elastic Stack
+
+As soon as elastic-package is set up, you need a running Elastic. The simplest thing to set it up with Docker is running `elastic-package stack up --version=8.13.0-SNAPSHOT -v`. This will start a snapshot build of an Elastic Stack cluster on your machine and you can open Kibana under `https://localhost:5601`.
+
+If you have your own Cloud setup or run Kibana from source, you can set the environment variables accordingly. The following is a sample assuming you are running Kibana and Elasticsearch from source: 
+
+```
+ELASTIC_PACKAGE_ELASTICSEARCH_HOST=http://localhost:9200
+ELASTIC_PACKAGE_ELASTICSEARCH_USERNAME=elastic
+ELASTIC_PACKAGE_ELASTICSEARCH_PASSWORD=changeme
+ELASTIC_PACKAGE_KIBANA_HOST=http://localhost:5601
+#ELASTIC_PACKAGE_CA_CERT
+```
+
+The same can be use to connect to your Elastic Cloud cluster. Unfortunately elastic-package currently does not support an API key to be used. 
+
+### Checkout integrations repository
+
+To ingest data for an integration, currently you have to checkout the integrations repo to your machine as the templates for the data are not part of the packages itself. Clone the integrations repo to your disk:
+
+```
+git clone https://github.com/elastic/integrations.git
+```
+
+### Ingest data
+
+Now jump to one of the packages which have a template file. We will be using nginx for this example but k8s and aws package also have templates inside. You need to `cd` into the nginx package:
+
+```
+cd packages/nginx
+```
+
+Inside the directory, run the following command with the `elastic-package` stack setup.
+
+```
+elastic-package benchmark stream -v
+```
+
+If I run I run Kibana from source, I run the following command:
```

Comment:
If Kibana starts with a basepath (default behavior with `yarn start`) or a `server.basePath` is set in the config, the `ELASTIC_PACKAGE_KIBANA_HOST` variable should be updated to include the path. If the env variable is left as is the issue is not really obvious from the `elastic-package` output.

Reply:
Good point. I'm so used to start Kibana with --no-base-path I missed it. I need to test if it works when the base path is added to the env variable or @gbamparop did you already try it?

Reply:
Tried it out and added an example.